pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+<!-- Please add the WIP label if the PR isn't complete. -->
+
+**Reviewer:** 
+
+<!-- Optional fields
+**CC:**
+**Depends on:** 
+-->
+
+## Description:
+
+Project task: 
+
+Relevant products which you have tested these changes against (i.e. extension, Windows, ...):
+- 
+
+Product PRs to test against:
+- 
+
+**Note: Upon merging this PR, you are responsible for updating the affected products to the latest version of this repo.**
```

Comment:
I'm afraid that this will make sure that people don't contribute edge-cases etc. IMO all products should automatically pull and run changes and if test is correct (and this is up to person creating PR and feature DRI and/or us) then failing test on some platform is like a bug report for them to align.